src/components/views/MainPage/MainPage.tsx

```diff
@@ -0,0 +1,42 @@
+import {Text, View} from 'react-native';
+import Logo from '../../../static/imgs/rst_logo.svg';
+import {styles} from './style';
+import {BackArrow, LoginForm, Main, RegisterForm} from '../../../components';
+import {createNativeStackNavigator} from '@react-navigation/native-stack';
+import {useNavigation} from '@react-navigation/native';
+import {globalStyles} from '../../../utils/globalStyles';
+import {useState} from 'react';
+import {ROUTES, RootStackParamList} from '../../../static/types/routeTypes';
+import {StackNavigationProp, createStackNavigator} from '@react-navigation/stack';
+
+const Tab = createNativeStackNavigator();
+
+export const MainPage = () => {
+  const [index, setIndex] = useState(0);
+  const navigation = useNavigation<StackNavigationProp<RootStackParamList>>();
+
+  return (
+    <View style={styles.main}>
+      {index !== 0 && <BackArrow goBack={() => navigation.navigate(ROUTES.MAIN)} />}
+      <Logo width={340} height={75} />
+      <Text style={styles.logoSubText}>Dla ludzi i innych stworów</Text>
+      <View style={globalStyles.navigationWrapper}>
+        <Tab.Navigator
```

Comment:
Czemu to jest w MainPage? W sensie cała nawigacja w jednym miejscu powinna być

Główne ekrany to tab'y, w taback są stacki

Reply:
w sensie? nie rozumiem, to jest po to żeby nie zmieniać całego componentu i nie rerenderowało się logo tylko samo content pod spodem